### PR TITLE
Update Issue Template

### PR DESCRIPTION
**Pull request by jens1o**
**Regarded to #134 **

**Changes proposed:**
* [X] Add
* [X] Prepare
* [ ] Delete
* [ ] Fix

**Things I've done:**
* [X] I've pulled the latest master branch
* [ ] I've run `npm install` to install all the dependenies
* [X] I've checked my code, it is clean.
* [ ] I've run ESLint.
* [X] My pull request fixes an issue, I referenced the issue.
* [X] My pull request is not too big, otherwise I've already squashed it.
* [ ] I've run `npm run build` to build the extension. (If I had done something with the extension.)

Adds/Prepares the new template. Feel free to correct me.

### CODE REVIEW

**Comment by jens1o** [comment]
Don't merge yet!

**Comment by jens1o** [comment]
You can now correct my mistakes, add something, discuss with me or just merge my request. :wink: I'm done.

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -16,10 +16,19 @@ _**Instruction Template**_
 - [ ] I've regarded to the [FAQ](https://github.com/robertohuertasm/vscode-icons/blob/master/README.md#faqs) and they did not help.
 - [ ] My system is: [OS] [Arch]
 - [ ] I'm sure this issue is not a *duplicate*?
+- [ ] I want to create an icon request(remove these lines under there if not):
```
Comment (robertohuertasm) [inline .github/ISSUE_TEMPLATE.md]: replace (if not, remove these lines below)

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -16,10 +16,19 @@ _**Instruction Template**_
...
+      - Icon Name: [name]
+      - Sample original Icon(32x32/png/transparent whenever possible): [url]
+      - Extensions: .just, .a, .bunch, .of, .extensions
```
Comment (robertohuertasm) [inline .github/ISSUE_TEMPLATE.md]: Nice!

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (jens1o) [inline .github/ISSUE_TEMPLATE.md]: okey.

**Comment by jens1o** [comment]
something else?

**Pull request merged**